system/classes/KO7/REST/Format/XML.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Class for formatting REST-Response bodies as XML
+ *
+ * @package KO7\REST
+ *
+ * @copyright  (c) since 2016 Koseven Team
+ * @license    https://koseven.ga/LICENSE
+ */
+class KO7_REST_Format_XML extends REST_Format {
+
+    /**
+     * Format function
+     *
+     * @param array $body Body to format
+     *
+     * @throws REST_Exception
+     *
+     * @return string
+     */
+    public function format(array $body): string
+    {
+        // Check if php-xml is loaded
+        if ( ! extension_loaded('xml'))
+        {
+            throw new REST_Exception('PHP XML Module not loaded.');
+        }
+
+        // Create new XML Element
+        $xml = new SimpleXMLElement('<root/>');
```

Comment:
use power of PHP 7.3

Reply:
I did not find another option to do this. SimpleXMLElement is the php-xml extension....don't know/think that there is a native function

Reply:
`$xml = new SimpleXMLElement('<?xml version="1.0" encoding="'.KO7::$charset.'"?><root/>', LIBXML_BIGLINES | LIBXML_COMPACT | LIBXML_NOCDATA | LIBXML_NOENT);`
also u can add  `| LIBXML_NOERROR | LIBXML_NOWARNING` to options and check `libxml_get_errors()` for throw exception

Reply:
Thanks, done and added error check / throw exception